lib/protocol/protocol.js

```diff
@@ -296,7 +296,7 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
 
       // if the status is not 0,  throw the appropriate error for status code.
       if (util.hasValue(driverRes)) {
-        if (util.hasValue(driverRes.status) && parseInt(driverRes.status, 10) !== 0) {
+        if (util.hasValue(driverRes.status) && parseInt(driverRes.status, 10) !== 0 && !_.isNaN(driverRes.status)) {
```

Comment:
Use built-in javascript isNaN function rather than lodash one. Also, this check should be placed before `parseInt`